types/oooRequest.d.ts

```diff
@@ -26,6 +26,24 @@ export type OooStatusRequestBody = {
   updatedAt?: Timestamp;
 };
 
+export type OooRequestUpdateBody = {
+  lastModifiedBy?: string;
+  type?: string;
+  id?: string;
+  reason?: string;
+  state: REQUEST_STATE.APPROVED | REQUEST_STATE.REJECTED;
+  updatedAt?: admin.firestore.Timestamp;
+};
```

Comment:
Is everything optional except state? 
Also, type had specific strings as types in design docs I think.

Reply:
its depend on the usage for middleware state is mandatory, for model updatedAt is optional.

Reply:
Design doc mentions `type` property to have type as specific string only.

![Screenshot_8](https://github.com/Real-Dev-Squad/website-backend/assets/87164140/e1b55819-8cc5-4a8b-87f3-f264d185d598)

Reply:
added